Makefile

```diff
@@ -335,26 +335,58 @@ shadow-server: export TARGET := clojure
 shadow-server:##@ Start shadow-cljs in server mode for watching
 	yarn shadow-cljs server
 
-test-watch: export TARGET := clojure
-test-watch: ##@ Watch tests and re-run no changes to cljs files
-	yarn install
-	nodemon --exec 'yarn shadow-cljs compile mocks && yarn shadow-cljs compile test && node --require ./test-resources/override.js target/test/test.js' -e cljs
+_clojure-test: export TARGET := clojure
+_clojure-test:
+	yarn install && \
+	yarn shadow-cljs compile mocks && \
+	yarn shadow-cljs compile test && \
+	node --require ./test-resources/override.js "$$SHADOW_OUTPUT_TO"
 
-test-watch-for-repl: export TARGET := clojure
-test-watch-for-repl: ##@ Watch tests and support REPL connections
-	yarn install
-	rm -f target/test/test.js
-	concurrently --kill-others --prefix-colors 'auto' --names 'build,repl' \
-		'yarn shadow-cljs compile mocks && yarn shadow-cljs watch test --verbose' \
-		'until [ -f ./target/test/test.js ] ; do sleep 1 ; done ; node --require ./test-resources/override.js ./target/test/test.js --repl'
+_clojure-test-watch: export TARGET := clojure
+_clojure-test-watch:
```

Comment:
Why does this start with underscore? And it's missing a help message.

Reply:
And should be called `test-clojure-watch` or similar to have the same prefix as other test targets.

Reply:
Also, why do we need a separate target? Why can't you just have a variable that decides if it's a watch version or not?

Reply:
I think it's a matter of convenience @jakubgs. Passing parameters to make is not as convenient as using the auto-complete of available targets and just pressing `tab` and `enter`. Obviously, this leads to a bit of code duplication.

We can have a variable for sure, a matter of taste :)

Reply:
Resolved by https://github.com/status-im/status-mobile/pull/18304/commits/bcf0af0a8544af4564e510affb8a50969015cf04

Now the dev must pass `WATCH=true` to the test targets they care about watching files and the Makefile is more DRY now.